src/TLuaInterpreter.cpp

```diff
@@ -4750,18 +4749,18 @@ int TLuaInterpreter::performHttpRequest(lua_State *L, const char* functionName,
         file.close();
     }
 
-    host.updateProxySettings(host.mLuaInterpreter.mpFileDownloader);
+    host.updateProxySettings(host.getNetworkAccessManager());
 
     QNetworkReply* reply;
     switch (operation) {
         case QNetworkAccessManager::PostOperation:
-            reply = host.mLuaInterpreter.mpFileDownloader->post(request, fileToUpload.isEmpty() ?dataToPost.toUtf8() : fileToUpload);
+            reply = host.getNetworkAccessManager()->post(request, fileToUpload.isEmpty() ?dataToPost.toUtf8() : fileToUpload);
```

Comment:
I'm surprised that CodeFactor hasn't picked up on a missing space here and further on below:
```suggestion
            reply = host.getNetworkAccessManager()->post(request, fileToUpload.isEmpty() ? dataToPost.toUtf8() : fileToUpload);
```